src/components/LoginForm.jsx

```diff
@@ -0,0 +1,26 @@
+Nameimport "../styles/login.css";
+
+export default function LoginForm(){
```

Comment:
I would suggest you to use arrow functions.

Reply:
will use them in a later commit 👍